src/structure/grid.ts

```diff
@@ -144,15 +144,15 @@ export class ViewersGrid {
      * Create a new [[ViewersGrid]] inside the HTML element with the given
      * `id`
      *
-     * @param id           HTML id of the DOM element where the viewer should live
+     * @param element      HTML element or HTML id of the DOM element where the name should live
```

Comment:
copy-paste error (name => structure grid) and same comment as above ("HTMLElement or string id ...")